CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ if (BUILD_PKI_CORE OR BUILD_PKI_CONSOLE)
     find_package(NSS REQUIRED)
 endif ()
 
-find_package(Java REQUIRED)
+find_package(Java COMPONENTS Development)
```

Comment:
@edewata I'm not quite sure why this change is necessary to build with JDK11+ on Fedora Rawhide. If you leave this as required, it complains about `javah` not being present. However a minimal test case with just CMake on the same machine doesn't have this problem (`REQUIRED` works just fine). I'm wondering if it has to do with the `include(Java)` above?

Reply:
I don't know the answer to that. However, I don't think we really need to build with JDK 11. If JDK 8 is available, we can continue building with it since the binaries will be compatible with JDK 11. If we have to use JDK 11, we probably need to specify some parameters to make sure the binaries are compatible with JDK 8.

Reply:
So the only thing we should _need_ to switch is `java_home` in the spec file; that's what I've been testing with. 

On the other hand, I think that we can actually drop this now; I'm unable to reproduce this and can build correctly without this commit. I _think_ the problem stemmed from exporting `JAVA_HOME` in the spec file versus changing the value of `java_home`.  :)